### ISSUE TITLE
Server invocation should pass in the desired port

### ISSUE THREAD

**Issue by abellotti** [abellotti/discovery-deployer#3]
For now we leverage the default 8000 from gunicorn here: https://github.com/abellotti/discovery-deployer/blob/master/discovery_template.yaml#L140

```
           command:
              - /bin/bash
              - '-c'
              - make server-migrate server-set-superuser -C /app &&
                python quipucords/manage.py runserver --nostatic
```

We need to update the server run script in the container to allow for alternate ports, essentially ending up with 
the gunicorn ....  --bind=0.0.0.0:${DISCOVERY_SERVER_PORT}   where our Template passes that DISCOVERY_SERVER_PORT appropriately.

**Comment by abellotti** [comment]
Now resolved with aec4b9d47665f8f89839743263cfffd21b19bc00

**Comment by abellotti** [comment]
This has been updated in favor of the QPC_SERVER_PORT env

See: https://github.com/abellotti/discovery-deployer/commit/f8e00452c8a75e2d93416bf2b879c81e09b86458
